inflation_radius: 0.08
cost_per_cell: 100
